src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -338,10 +338,11 @@ public function sendHeaders()
 
         // cookies
         foreach ($this->headers->getCookies() as $cookie) {
+            $path = $cookie->getPath().(null !== $cookie->getSameSite() ? ('; samesite='.$cookie->getSameSite()) : '');
```

Comment:
I understand that this is a hack needed because of the lack of support of `samesite` in PHP cookie functions. However, wouldn't it be better to hack this into  `$cookie->getDomain()` instead of `$cookie->getPath()` ?

Reply:
@javiereguiluz the issue is, domain can be null while path will always be something

Reply:
OK then 👍  ... but @nicolas-grekas proposal may be a better way to solve this? What do you think?